src/app.rs

```diff
@@ -501,13 +504,27 @@ impl MacroSolverApp {
                 );
                 ui.add(HelpText::new("Forces any action that increases Progress to only be used at the end of the rotation."));
             });
+            ui.horizontal(|ui| {
```

Comment:
Warning for each option should come right after the option itself. Please move the adversarial checkbox below the warning for progress backloading.